lib/galaxy/webapps/tool_shed/api/categories.py

```diff
@@ -80,6 +83,7 @@ def index( self, trans, deleted=False, **kwd ):
             category_dict[ 'url' ] = web.url_for( controller='categories',
                                                   action='show',
                                                   id=trans.security.encode_id( category.id ) )
+            category_dict[ 'repositories' ] = trans.app.repository_registry.viewable_repositories_and_suites_by_category.get( category.name, 0 )
```

Comment:
As discussed offline, this one only shows the count of repositories in the relevant category.